Makefile

```
@@ -173,15 +173,24 @@ run-front-for-puppeteer:
 # ----------------------------------------------------------
 # Remote: Run commands
 
-.PHONY: restart-remote-app ##RUN ▶️  Restart app
-restart-remote-app:
+prune-old-images: # Not using `docker image prune` because it would delete the data pipeline image if run when no flow is running
+	docker images --format "{{.Repository}}:{{.Tag}}" | \
+	awk -F: '{repos[$1]++; if(repos[$1]>1) print $0}' | \
+	xargs -r docker rmi
+
+restart-remote-app-without-image-prune:
 	cd infra/remote && docker compose pull && docker compose up -d --build app --force-recreate
 
-.PHONY: register-pipeline-flows-prod ##RUN ▶️  Register pipeline flows in PROD
-deploy-pipeline-flows:
+.PHONY: restart-remote-app ##RUN ▶️  Restart app
+restart-remote-app: restart-remote-app-without-image-prune prune-old-images
```

```suggestion
restart-remote-app: prune-old-images restart-remote-app-without-image-prune
```

Je propose d'inverser l'ordre pour garder la dernière release, au cas où la MEP actuelle fail et qu'on doive run l'ancienne image, wdyt ?